helm/hpcc/templates/sasha.yaml

```diff
@@ -28,6 +28,10 @@ spec:
     spec:
       {{- include "hpcc.placementsByPodTargetType" (dict "root" $ "pod" $serviceName "type" "sasha") | indent 6 }}
       serviceAccountName: "hpcc-default"
+      {{- if $sasha.storage }}
```

Comment:
sasha services also live in the Dali pod (see dali.yaml generation of sasha services),
so that too should check if any have storage and forcePermissions , and create an initContainer.

That's an edge case, because the only sasha service by default under Dali doesn't need storage, but you can change the hpcc values config so the ones that do are in dali pod.

Please open a separate JIRA to address this after this one.

Reply:
https://track.hpccsystems.com/browse/HPCC-25768 "Check sasha storage in dali.yaml" is created

Reply:
Update JIRA link: https://track.hpccsystems.com/browse/HPCC-25853 "Process sasha services inside dali for forcePermission:true" is created